### PR TITLE
Label formatting

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

File: src/main/resources/standardTestCases/labels/invalid.labels [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "0-label-key-1":"0-label-value-1",
-  "the-maximum-allowed-character-length-for-label-pairs-is-sixty-three": "cromwell-please-dont-validate-these-labels"
+  "the-maximum-allowed-character-length-for-label-pairs-is-sixty-three": "cromwell-please-dont-validate-these-labels",
+  "": "not a great label key"
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/labels/invalid.labels:4]: the worst, actually

File: src/main/resources/standardTestCases/labels/jes_labels.wdl [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+task My_task {
+  command {
+    sleep 1
+  }
+  runtime {
+    docker: "ubuntu:latest"
+    preemptible: 3
+  }
+}
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/labels/jes_labels.wdl]: \n

File: src/main/resources/standardTestCases/labels/jes_labels.wdl [hunk 3]
```diff
@@ -0,0 +1,13 @@
+workflow I_hope_nobody_names_workflows_like_this_as_it_seems_very_unnecessary_ {
+    call MY_TASK as my_task_aliased {}
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/labels/jes_labels.wdl:2]: wait is WDL not case-sensitive?  😦

**Review by mcovarr — COMMENTED** [review]
I'll save my thumb for when the tests pass 😄

File: src/main/resources/standardTestCases/jes_labels.test [hunk 4]
```diff
@@ -1,24 +1,28 @@
 name: jes_labels
 testFormat: workflowsuccess
 tags: [ labels ]
-backends: [Jes]
+ignore: true
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/jes_labels.test]: ummmm 😦

File: src/main/resources/standardTestCases/labels/jes_labels.wdl (hunk 3 above)
Comment (ruchim) [reply src/main/resources/standardTestCases/labels/jes_labels.wdl:2]: it is!!!!!!!! this PR was probably opened too soon :speak_no_evil:

**Comment by cjllanwarne** [comment]
Presumably these are deterministic once the Cromwell PR is approved, so 👍 with that in mind (and after @mcovarr's newline is added!)

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/217/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by mcovarr** [comment]
:+1: with the whole test-passing thing 😛 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/217/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**